packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart

```diff
@@ -120,8 +120,8 @@ class ResidentWebRunner extends ResidentRunner {
       return printHelpDetails();
     }
     const String fire = '🔥';
-    const String rawMessage =
-        '  To hot restart (and rebuild state), press "r" or "R".';
+    const String rawMessage = '  To hot reload changes while running, press "r". '
+      'To hot restart (and rebuild state), press "R".';
```

Comment:
By reading this, I still don't understand the difference between "r" and "R". I think one major source of confusion for us (web developers) is that "reload" has a different meaning from Flutter's reload. In web, reload means refresh the page.

![image](https://user-images.githubusercontent.com/1278212/67048659-71df0c00-f0e9-11e9-8f72-2d4a7fb65ec0.png)

Reply:
What suggestions do you have to further disambiguate them? In conventional flutter, restart seems similar to reloading the page, and "hot reload" is a known concept as well.

Reply:
We could call restart = refresh?

Reply:
"hot restart" and "page reload"?

Reply:
i.e. 'To hot restart, press "r". To reload the page, press "R".'

Reply:
I like what @yjbanov suggested. But looking at it from Flutter's perspective, users are used to hitting "R" for "hot restart", now they will be hitting "r" for "hot restart".

Reply:
Hot restart today on mobile devices restarts the entire app more or less. Hot restart on the web is sort of a "someday it will be hot reload"